src/contracts/core/RewardsCoordinator.sol

```diff
@@ -340,6 +363,83 @@ contract RewardsCoordinator is
         }
     }
 
+    /**
+     * @notice Validate a PerformanceRewardsSubmission. Called from `createAVSPerformanceRewardsSubmission`.
+     * @param performanceRewardsSubmission PerformanceRewardsSubmission to validate.
+     * @return total amount to be transferred from the avs to the contract.
+     */
+    function _validatePerformanceRewardsSubmission(
```

Comment:
Not checking for `MAX_FUTURE_LENGTH` (Since Performance based reward submissions are retroactive) or `MAX_REWARDS_AMOUNT` (Since we no longer have the `1e38 - 1` limitation in the offchain calculation.

Reply:
could u add this in code comments?

Reply:
Done in [179aa1f](https://github.com/Layr-Labs/eigenlayer-contracts/pull/837/commits/179aa1fb13d8d8d0aede96e87173978bfc0a5c62) as a `@dev` natspec.